docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,110 @@
+# Getting started
+
+Welcome to the Lightning Development Kit documentation!
+
+If you have any questions about anything related to LDK, feel free to ask our community on [GitHub Discussions](https://github.com/orgs/lightningdevkit/discussions) or join us on [Discord](https://discord.gg/5AcknnMfBw). 
+
+## System Requirements
+MacOS, Windows and Linux are supported.
+
+## Installation
+To add LDK to a project, run:
+
+:::: tabs
+::: tab "Rust"
+```rust
+/*
+Add the following dependencies to your cargo.toml and replace {VERSION} with the 
+version number you want to use from
```

Comment:
Little cleaner if you drop "from". Likewise elsewhere.

Reply:
Fixed [03764a8](https://github.com/lightningdevkit/lightningdevkit.org/pull/133/commits/03764a8334a6940e905a41d80753400f9e659e71)